docs/guide/slots.md

```diff
@@ -51,9 +55,13 @@ To render a `renders_many` slot, iterate over the name of the slot:
 <% end %>
 ```
 
+The link to "My blog" will render inside the `<h1>` tags, and links to each `BlogPost` will be rendered beneath.
```

Comment:
I wonder if it would be simpler to just put what HTML this example renders here instead of describing it in words.

```suggestion
The link to "My blog" will render inside the `<h1>` tags, and links to each `BlogPost` will be rendered beneath.
```

Reply:
Agreed, I'll make the change.